.github/workflows/release.yml

```diff
@@ -59,3 +59,17 @@ jobs:
           prerelease: "true"
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      # Allow to release Epinio Helm chart automatically when we release Epinio.
+      # The latest tag is sent to the Helm chart repo.
+      - name: Get the last tag
+        id: last-tag
+        run: |
+          tag=$(git -P tag | tail -1)
```

Comment:
I wonder if there is an edge case where this behaves different from the "git describe --tags" used by the Makefile.

Reply:
If I'm not wrong, `git describe --tags`  gives also the commit SHA.
Something like:
```
git describe --tags
epinio-0.1.1-3-g87b21da
```

Reply:
Only if you're not directly on the tag:

```
% git checkout v0.1.4
% git describe --tags
v0.1.4
```